api/src/routes/category-list.ts

```diff
@@ -22,6 +22,9 @@ export async function categoryListHandler(req: Request, res: Response) {
       where: (category, { eq }) => eq(category.shopId, user.shopId),
     })
 
+    // cache for 1 week
+    res.set("Cache-Control", "public, max-age=604800")
```

Comment:
These data endpoints almost never changes, so it's a great candidate for caching. The frontend calls this endpoint with a `?site=...` query parameter, so the cache is isolated by site.